src/styles/_components/_status-tags.scss

```diff
@@ -0,0 +1,22 @@
+/*
+   status-tag
+    - example: https://projects.invisionapp.com/d/main/default/#/console/10009152/215179492/preview : expired and pending status
+    - the new colors should be added to the uniq colors
+*/
+.status-tag {
+  @include font("IntervalNextMed");
+  border-radius: 2px;
+  font-size: em(14px);
+  padding: 2px 9px 4px 9px;
+  display: inline-block;
```

Comment:
opt for `display: block;` instead as the placement of this component will be decided on a per-platform-basis.